public/assets/css/vendors.min.css

```
@@ -20369,6 +20369,727 @@
+.ts-wrapper {
+    position: relative;
+}
+
+.ts-dropdown,
+.ts-control,
+.ts-control input {
+    color: #343a40;
+    font-family: inherit;
+    font-size: inherit;
+    line-height: 1.5;
+    font-smoothing: inherit;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)